cli/flox/src/commands/edit.rs

```diff
@@ -280,12 +279,15 @@ impl Edit {
         match result {
             Err(EnvironmentError::Core(e @ CoreEnvironmentError::LockedManifest(_)))
             | Err(EnvironmentError::Core(e @ CoreEnvironmentError::DeserializeManifest(_)))
-            | Err(EnvironmentError::Core(
-                e @ CoreEnvironmentError::BuildEnv(CallPkgDbError::PkgDbError(PkgDbError {
-                    exit_code: error_codes::BUILDENV_CONFLICT,
-                    ..
-                })),
-            )) => Ok(Err(e)),
+            // TODO: detect path conflict with in builenv.nix output!
+            //
+            // | Err(EnvironmentError::Core(
+            //     e @ CoreEnvironmentError::BuildEnv(CallPkgDbError::PkgDbError(PkgDbError {
+            //         exit_code: error_codes::BUILDENV_CONFLICT,
+            //         ..
+            //     })),
+            // ))
```

Comment:
Oh I forgot about all the pkgdb error handling. We should probably check if we lost anything important...